src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,54 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+interface Props {
+  select: (event: React.ChangeEvent<HTMLSelectElement>) => void,
+  search: (event: React.ChangeEvent<HTMLInputElement>) => void,
+  value: string,
+  clearSearch: () => void,
+}
```

Comment:
```suggestion
interface TodoFilter {
  select: (event: React.ChangeEvent<HTMLSelectElement>) => void,
  search: (event: React.ChangeEvent<HTMLInputElement>) => void,
  value: string,
  clearSearch: () => void,
}
```

checkout checklist 3rd point about naming conventions for props